tests/installation/welcome.pm

```diff
@@ -111,6 +111,20 @@ sub run {
         $self->accept_license;
     }
 
+    # Verify install arguments passed by bootloader
+    # Linuxrc writes its settings in /etc/install.inf
+    if (get_var('NETBOOT') && !check_var('BACKEND', 'ipmi')) {
+        wait_screen_change { send_key 'ctrl-alt-shift-x' };
+        my $method     = uc get_required_var('INSTALL_SOURCE');
+        my $mirror_src = get_var("MIRROR_$method");
+        die "Install source mismatch in boot parameters!\n"
+          if (script_run 'grep -o --color=always install=' . $mirror_src . ' /proc/cmdline');
```

Comment:
Let's add comparison to `0` exit code, as `script_run` is confusing as returns bash false which we interpret as perl true value. WDYT?

Reply:
Updated, thanks for the point! Just wondering if I should add ```$rc=undef``` between the conditions. WDYT ?

Reply:
Not really, as script_run will override previous value unconditionally. Btw, with your current code you could use one liner for the `die`, as you use the `$rc` variable ;)

Reply:
Ah, I could realize that :D